.docheader

```diff
@@ -3,3 +3,4 @@
  * @copyright Copyright (c) %regexp:(20\d{2}-)?20\d{2}% Zend Technologies USA Inc. (http://www.zend.com)
  * @license   https://github.com/zendframework/zend-expressive-router/blob/master/LICENSE.md New BSD License
  */
+declare(strict_types=1);
```

Comment:
IIRC, we need one empty line between the file-level docblock and any directives, per PSR-12. I'll update those when I merge.